MapboxNavigation/ErrorCode.swift

```diff
@@ -19,3 +19,18 @@ public enum MapboxNavigationError: Int {
      */
     case noDataInSpokenInstructionResponse
```

Comment:
Rename `noDataInSpokenInstructionResponse` to `emptyAwsResponse`, since it’s assumed that any case in this enumeration is about spoken instructions.

Previous iterations of this PR also had error codes for `noAwsResult` and `audioPlayerFailedToPlay`. Do we still need them?

Reply:
Yes, `.spokenInstructionAudioPlayerFailedToPlay` are still used.